utils/slackbots/suggestion-bot/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "apphome-demo-note",
+  "version": "0.0.1",
+  "description": "Stickie note app to demonstrate App Home feature.",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "dependencies": {
+    "axios": "^0.19.2",
+    "body-parser": "^1.19.0",
```

Comment:
This might have helped https://api.slack.com/events-api

https://api.slack.com/events/message.channels
https://api.slack.com/events/message.groups

![image](https://user-images.githubusercontent.com/47481188/87505525-ed8ccb80-c61d-11ea-9cfc-cb21a5691d42.png)

Reply:
Thanks for pointing me to a few more specific pages! I was reading the general API page and understood that she was checking against those events. I still didn't get from this for certain that starting up some sort of Express server was unnecessary, but I'll trust that it is. Feel free to ignore this comment as well.